parachain/pallets/ethereum-beacon-client/src/merkleization.rs

```diff
@@ -276,6 +258,22 @@ pub fn hash_tree_root<T: SimpleSerializeTrait>(mut object: T) -> Result<[u8; 32]
     }
 }
 
+pub fn get_sync_committee_bits(bits_hex: Vec<u8>) -> Result<Vec<u8>, MerkleizationError> {
+    let bitv = Bitvector::<{ config::SYNC_COMMITTEE_SIZE }>::deserialize(&bits_hex).map_err(|_| MerkleizationError::InvalidLength)?;
+
+    let mut result = Vec::new();
+
+    for bit in bitv.iter() {
+        if bit == true {
```

Comment:
Style: you can map over `bitv` and return a `vec` instead of using mutable `vec`.

Reply:
Addressed in [sync_committee_bits](https://github.com/Snowfork/snowbridge/pull/638/commits/8fdc1a156c72137a784def385d575baeb5483258).